### PR TITLE
fix some minor warnings

### PR DESCRIPTION
**Pull request by zhaonian**

### CODE REVIEW

File: JetNews/app/src/main/java/com/example/jetnews/ui/article/ArticleScreen.kt [hunk 1]
```diff
@@ -104,7 +104,7 @@ private fun BottomBar(post: Post, onUnimplementedAction: () -> Unit) {
                         onBookmark = { toggleBookmark(postId = post.id) }
                     )
                     BottomBarAction(R.drawable.ic_share) {
-                        SharePost(post, context)
+                        sharePost(post, context)
```
Comment (nickbutcher) [inline JetNews/app/src/main/java/com/example/jetnews/ui/article/ArticleScreen.kt]: By convention we capitalize composable functions, please revert this.

Comment (zhaonian) [reply JetNews/app/src/main/java/com/example/jetnews/ui/article/ArticleScreen.kt]: `sharePost` is not a composable function.

Comment (nickbutcher) [reply JetNews/app/src/main/java/com/example/jetnews/ui/article/ArticleScreen.kt]: Oh sorry!

Comment (zhaonian) [reply JetNews/app/src/main/java/com/example/jetnews/ui/article/ArticleScreen.kt]: yayy! first PR in this repo lol. thx!

**Comment by nickbutcher** [comment]
Thanks for the contribution! 🎉🎉🎉

**Pull request merged**